chainio/clients/fireblocks/contract_call.go

```diff
@@ -16,6 +17,10 @@ const (
 	Burn         TransactionOperation = "BURN"
 	TypedMessage TransactionOperation = "TYPED_MESSAGE"
 	Raw          TransactionOperation = "RAW"
+
+	FeeLevelHigh   FeeLevel = "HIGH"
+	FeeLevelMedium FeeLevel = "MEDIUM"
+	FeeLevelLow    FeeLevel = "LOW"
```

Comment:
fireblocks doesn't support setting a number for gasPrice?

Reply:
It does, and we should add them too in the future, but I wanted to just use the fees Fireblocks estimates for us given the fee level

Reply:
sgtm. Can you just add a TODO comment then to support those, unless its already written somewhere else?

Reply:
done